main.go

```diff
@@ -58,12 +59,20 @@ func main() {
 	}
 
 	var cmds []*exec.Cmd
+	// check for a .git directory and whether it's empty
+  if isDirEmpty(filepath.Join(c.Dir, ".git")) {
```

Comment:
there is an awesome plugin for Atom (and Sublime) that should auto-format your Go code every time you save. Otherwise you can manually run gofmt